spec/sec-class-definitions-patch.html

```diff
@@ -171,9 +156,31 @@ <h1>Static Semantics: VarScopedDeclarations</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-runtime-semantics-classstaticblockevaluation">
+    <h1><ins>Runtime Semantics: ClassStaticBlockEvaluation</ins></h1>
+    <p>With parameter _homeObject_.</p>
+    <emu-grammar collapsed>ClassStaticBlock: `static` `{` ClassStaticBlockBody `}`</emu-grammar>
+    <emu-alg>
+      1. Let _lex_ be the running execution context's LexicalEnvironment.
+      1. Let _privateScope_ be the running execution context's PrivateEnvironment.
+      1. Let _block_ be FunctionCreate(~Method~, &laquo; &raquo;, |ClassStaticBlockBody|, _lex_, _privateScope_).
+      1. Perform MakeMethod(_block_, _homeObject_).
+      1. Return the ClassStaticBlockDefinition Record { [[Block]]: _block_ }.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-runtime-semantics-classelementevaluation">
+    <h1>Runtime Semantics: ClassElementEvaluation</h1>
+    <p>With parameters _object_ and _enumerable_.</p>
```

Comment:
What's the _enumerable_ for?

Reply:
ClassElementEvaluation takes two parameters in the class fields proposal. Note that this is existing spec text (no `<ins>`), though I've elided the existing SDO algorithms for brevity to focus on what this proposal changes (the `<ins>` blocks on the following lines).